MoonAngleCondition/MoonAngleCondition.cs

```diff
@@ -74,6 +75,9 @@ public class MoonAngleCondition : SequenceCondition, IValidatable {
         public double SeparationLimit {
             get => separationLimit;
             set {
+                if (lorentzian) {
+                    value = LorentzianSeparation(value, 14);
```

Comment:
Will `width` ever need to be tunable?

Reply:
Yes, it's now specified when Lorentzian is enabled, with a default of 14